packages/webapp/components/Governance/ProposalCard.tsx

```diff
@@ -30,6 +30,7 @@ export const ProposalCard: FC<ProposalCardProps> = ({
   ...props
 }) => {
   const proposalFinalized = isFinalized(status)
+  // @ts-ignore
```

Comment:
why "ts-ignore" here?

Reply:
Left over from development...was trying to find the right balance between fast vs. perfect for MVP, but turns out I already fixed it so comment is not needed. Removing.